src/platforms/eglstream-kms/server/platform_symbols.cpp

```diff
@@ -44,6 +45,10 @@ namespace mgc = mir::graphics::common;
 namespace mo = mir::options;
 namespace mge = mir::graphics::eglstream;
 
+namespace
+{
+int const EGL_DRM_MASTER_FD_EXT{0x333C};
```

Comment:
Can you put this, adds context to the magic
```
/* XXX khronos eglext.h does not yet have EGL_DRM_MASTER_FD_EXT */
#if !defined(EGL_DRM_MASTER_FD_EXT)
#define EGL_DRM_MASTER_FD_EXT                   0x333C
#endif
```

Reply:
There's already a similar incantation in src/platforms/eglstream-kms/server/platform.cpp, so I would view this as pre-existing.

However, this code will break on any platform that #defines EGL_DRM_MASTER_FD_EXT. We should prevent that with an incantation similar to that @gerboland suggests. (I'd be happy to write or approve a follow-up PR fixing both instances.)

Reply:
Ok, will approve now then
bors r+